.gitignore

```diff
@@ -67,3 +67,5 @@ Temporary Items
 
 # End of https://www.toptal.com/developers/gitignore/api/clojure,leiningen,intellij,visualstudiocode,macos
 /test-results/
+.clj-kondo/
```

Comment:
The config.edn file in this directory should not be ignored though, should it?

Reply:
If you were to configure clj-kondo in the root directory then you would have to change the .gitignore accordingly. As it is, it's only configured in the core directory, and this changes nothing here. I'm happy to remove the change, it's really no bother to if you'd prefer, whatever makes you comfortable.

Reply:
Hmm, are you sure about that? I would have thought you'd need to use a specific path like `/.clj-kondo/` to exclude only that, and if you use only a name, like `.clj-kondo/` it would apply to any file/directory with that name.

Reply:
Yep, you're right of course, my mistake: I should have used a leading slash. However, the change is really not at all important to the issue or PR and so it's neater to just back it out, which I've now done. Thanks!

Reply:
@anderseknert - sorry I've just found a problem, please hold on the merge. By definition, the `build/sourceDirectory` in`pom.xml` contains one entry, but the `:source-paths` in `project.clj` requires two entries: `["src/main/clj" "src/main/cljc"]`. Lein transcribes the first entry only, and puts the other entries in the plugin section which deps does not scan, apparently. The outcome: only the clj files are on the class path. I'm going to swap the order and test now.

Reply:
So, changing the order just resulted in the opposite effect, and I'm still unable to load `jarl.eval` or `jarl.parse` without missing namespaces. However, because the `cljc` is not AOT-compiled, we can add it to the `:resource-paths` which does support multiple entries in the POM, and is loaded by deps. In short, the latest commit works now through adding `src\main\cljc` to `:resource-paths`, with all tests still passing.

Reply:
Thanks! LGTM. @johanfylling any thoughts?

Reply:
No comments other than: this is great! 🙂